jvb/src/main/kotlin/org/jitsi/videobridge/message/BridgeChannelMessage.kt

```diff
@@ -372,3 +377,15 @@ class RemoveReceiverMessage(
         const val TYPE = "RemoveReceiver"
     }
 }
+
+class BandwidthAllocationSettingsMessage(
```

Comment:
I don't love the name `BandwidthAllocationSettings` for this - from a client's point of view, it's not really talking about bandwidth allocation, it's talking about video reception preferences or something like that.